azavea/iow-boundary-tool
# File upload UI updates

## Overview

This PR updates the UI of File Upload section of the Welcome modal. This includes adding the ability to remove uploaded files.

Closes #191 

### Demo

<img width="1033" alt="Screenshot 2022-11-11 at 9 51 14 AM" src="https://user-images.githubusercontent.com/8356789/201380229-9eb3450c-5a64-49e7-8c9d-ad44bf29a815.png">
<img width="1030" alt="Screenshot 2022-11-11 at 9 53 04 AM" src="https://user-images.githubusercontent.com/8356789/201380230-a800fb36-9a62-4998-b2a6-2d969df7c971.png">


## Testing Instructions

- http://localhost:4545
- Login as c1@azavea.com
- Select other utility
- Proceed through welcome modal
  - [x] Ensure file upload section looks like the figma diagram
- Upload some files
  - [x] Ensure they show up in on modal
  - [x] Ensure they are removable with the trash can icon

## Checklist
